test/common/network/connection_impl_test.cc

```diff
@@ -242,8 +246,8 @@ class ConnectionImplTest : public testing::TestWithParam<Address::IpVersion> {
     auto transport_socket = std::make_unique<NiceMock<MockTransportSocket>>();
     EXPECT_CALL(*transport_socket, canFlushClose()).WillRepeatedly(Return(true));
 
-    return ConnectionMocks{std::move(dispatcher), timer, std::move(transport_socket), file_event,
-                           &file_ready_cb_};
+    return ConnectionMocks{std::move(dispatcher),       {},         timer,
+                           std::move(transport_socket), file_event, &file_ready_cb_};
```

Comment:
nit: unusual looking formatting, can you rerun format?

Reply:
Reran format.

Reply:
nit: will explicitly constructing `Server::MockThreadLocalOverloadState()`  make formatter happy?